.github/workflows/containers.yml

```diff
@@ -17,15 +17,44 @@ jobs:
         run: |
           docker build . --tag topo-imagery --label "github_run_id=${GITHUB_RUN_ID}"
 
-      - name: Log in to registry
+      - name: Log in to Registry
         if: ${{(github.ref == 'refs/heads/master') && !(startsWith(github.event.head_commit.message, 'release:'))}}
         run: echo "${{ secrets.GITHUB_TOKEN }}" | docker login ghcr.io -u $ --password-stdin
 
       - name: Publish Containers
```

Comment:
Should we name the step `Publish Containers to GHCR` and `Tag and push docker image to Amazon ECR` to `Publish Containers to ECR`?